helm/cert-exporter/templates/deployment/servicemonitor.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "cert-exporter.fullname" . }}-cert-manager
+  namespace: {{ template "cert-exporter.namespace" . }}
+  labels:
+    app: {{ template "cert-exporter.name" . }}-coredns
```

Comment:
Are you sure for "-coredns" suffixes? Seems, copy-paste

Reply:
You're absolutely right, good catch, thanks.